redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/base/DatabaseServerV4Identifiers.java

```diff
@@ -1,45 +1,27 @@
 package com.sequenceiq.redbeams.api.endpoint.v4.databaseserver.base;
 
+import static com.sequenceiq.redbeams.doc.ModelDescriptions.CRN;
+
 import java.io.Serializable;
 
 import javax.validation.constraints.NotNull;
-import javax.validation.constraints.Pattern;
-import javax.validation.constraints.Size;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import com.sequenceiq.redbeams.doc.ModelDescriptions;
-import com.sequenceiq.redbeams.doc.ModelDescriptions.DatabaseServer;
 
 import io.swagger.annotations.ApiModelProperty;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class DatabaseServerV4Identifiers implements Serializable {
```

Comment:
Since this class now only holds a single field, should it be eliminated?

Reply:
Yes, thank you!